councilmatic_core/models.py

```diff
@@ -179,7 +179,7 @@ class Bill(models.Model):
     identifier = models.CharField(max_length=50)
     bill_type = models.CharField(max_length=50)
     classification = models.CharField(max_length=100)
-    source_url = models.CharField(max_length=255)
```

Comment:
why can't we use the API url here. seems like that would make debugging easier in the future?

Reply:
yeah, I was waffling on this one, [since Pupa requires a `source_url`](Pupa requires a source
I cannot think of a solid reason to require a `source_url`)....seems like Councilmatic should too. 

If so, then I need to make a change in the scraper too, i.e., add [a note to the source_url.](https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/bills.py#L174)

Reply:
Looks like I already opened an issue with a question about api vs. web: https://github.com/datamade/django-councilmatic/issues/227